op_crates/file/01_file.js

```diff
@@ -300,42 +344,62 @@
     /**
      * @param {BlobPart[]} fileBits 
      * @param {string} fileName 
-     * @param {FilePropertyBag} [options] 
+     * @param {FilePropertyBag} options 
```

Comment:
```suggestion
     * @param {FilePropertyBag=} options 
```

Reply:
Results in `options` is possibly undefined, which is wrong.